Userland/Libraries/LibJS/Runtime/TypedArray.cpp

```diff
@@ -143,6 +143,12 @@ static void initialize_typed_array_from_array_like(GlobalObject& global_object,
     if (vm.exception())
         return;
 
+    // Enforce 2GB "Excessive Length" limit
+    if (length > NumericLimits<i32>::max() / sizeof(TypeError)) {
```

Comment:
not sure I understand this, why `sizeof(TypeError)`?

Reply:
Oops! i meant `Type`, fixed!